src/schema/FederationList.json

```diff
@@ -27,6 +27,11 @@
                           "default": false,
                           "example": false
                                         },
+                       "ik": {
+                         "description": "The IK (Institutionskennzeichen) numbers associated with this domain. Must contain at least one element when isInsurance is true. Must not contain any elements when isInsurence is false.",
+                         "type": "array",
+                         "items": { "type": "number" }
```

Comment:
I'm not sure if `number` is correct here. At least the values in the [spec](https://gemspec.gematik.de/docs/gemSpec/gemSpec_VZD_FHIR_Directory/latest/#4.2.3) are not strings.

Reply:
Probing the test environment unveils that string values are silently converted to numbers. So `number` is likely correct here.

![Screenshot 2025-01-30 at 15 22 50](https://github.com/user-attachments/assets/91cb3cd6-fabb-4aa6-b8e4-307e92252f7c)

Reply:
Probing the test environment unveils that string values are silently converted to numbers. So `number` is likely correct here.

![Screenshot 2025-01-30 at 15 22 50](https://github.com/user-attachments/assets/91cb3cd6-fabb-4aa6-b8e4-307e92252f7c)

Reply:
Probing the test environment unveils that string values are silently converted to numbers. So `number` is likely correct here.

![Screenshot 2025-01-30 at 15 22 50](https://github.com/user-attachments/assets/91cb3cd6-fabb-4aa6-b8e4-307e92252f7c)